packages/coinstac-simulator/src/file-loader.js

```diff
@@ -0,0 +1,110 @@
+'use-strict';
+
+const glob = require('glob');
+const loadCSV = require('csv-load-sync');
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+
+/**
+ * syncronously loads and parses files from a globbing pattern, and concats the data
+ * into an array
+ * @param  {string} fileGlob glob pattern for input files
+ * @param  {string} delim    delimiter used for txt files, defualts to ','
+ * @return {array}          array of parsed input data
+ */
+const loadFiles = (fileGlob, delim) => {
+  const delimiter = delim || ',';
+  const files = glob.sync(fileGlob);
+  let userData = [];
+  files.forEach((filePath) => {
+    if (fs.lstatSync(filePath).isFile()) {
+      const data = fs.readFileSync(filePath, 'utf8');
+      const ext = path.extname(filePath);
+      if (ext === '.json') {
+        // expects JSON array
+        userData = userData.concat(JSON.parse(data));
```

Comment:
This all just concats the arrays together for one 'variable', it might? be nice to have a 'mode' option to do other operations on multiple files passed into one variable. However it may not be a usual case to have multiple files per var, unsure.